test/jobs/refresh_resource_quota_utilization_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_plugin_helper'
+require 'foreman_tasks/test_helpers'
```

Comment:
I guess you should fix rubocop ;) 

How is the test related to the permission fix?